### PR TITLE
added example to ones_like

### PR DESCRIPTION
**Pull request by soniasalomoni**
----
:books: Documentation preview :books:: https://equistore--205.org.readthedocs.build/en/205/

### CODE REVIEW

File: python/src/equistore/operations/ones_like.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from typing import List, Union
+import numpy as np
```
Comment (agoscinski) [inline python/src/equistore/operations/ones_like.py:2]: The linter seems not to be soo smart, so it throws an error here
https://github.com/lab-cosmo/equistore/actions/runs/4417733640/jobs/7745861938#step:7:532
In the code you don't need numpy, so it complains. It apparently cannot see that it is used in the doc.
This should fix the problem
```suggestion
import numpy as np # noqa: F401
```

File: python/src/equistore/operations/ones_like.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
 
-from equistore import TensorBlock, TensorMap
+from equistore import TensorBlock, TensorMap, Labels
```
Comment (agoscinski) [inline python/src/equistore/operations/ones_like.py:4]: Same issue and fix as above
https://github.com/lab-cosmo/equistore/actions/runs/4417733640/jobs/7745861938#step:7:533

File: python/src/equistore/operations/ones_like.py [hunk 3]
```diff
@@ -46,6 +160,9 @@ def ones_like_block(
                        If empty list ``[]`` no gradients information are copied.
     :param requires_grad: If autograd should record operations for the returned tensor.
                           This option is only relevant for torch.
+
+
+
```
Comment (agoscinski) [inline python/src/equistore/operations/ones_like.py:165]: Nothing super important, but could you remove these empty lines?

**Review by agoscinski — COMMENTED** [review]
Thanks!

**Comment by DavideTisi** [comment]
to make the linter happy talking to @Luthaf we thought of adding the import in the example in the docsstring

File: python/src/equistore/operations/ones_like.py (hunk 1 above)
Comment (agoscinski) [reply python/src/equistore/operations/ones_like.py:2]: Okay talked with @DavideTisi and @Luthaf we move this import to the docs, so you can remove the import here and insert it into the docstring
```suggestion
```

File: python/src/equistore/operations/ones_like.py (hunk 2 above)
Comment (agoscinski) [reply python/src/equistore/operations/ones_like.py:4]: same here we remove the Labels import

File: python/src/equistore/operations/ones_like.py [hunk 4]
```diff
@@ -20,6 +21,119 @@ def ones_like(
                        If empty list ``[]`` no gradients information are copied.
     :param requires_grad: If autograd should record operations for the returned tensor.
                           This option is only relevant for torch.
+
+    Here is an example using this function:
+
+    First we create a ``TensorMap`` with just one block with two gradients, named ``alpha`` and `beta`.
+
```
Comment (agoscinski) [inline python/src/equistore/operations/ones_like.py:28]: and add the import here
```suggestion
    >>> import numpy as np
    >>> from equistore import Labels
```

File: python/src/equistore/operations/ones_like.py (hunk 2 above)
Comment (agoscinski) [reply python/src/equistore/operations/ones_like.py:4]:
```suggestion
from equistore import TensorBlock, TensorMap
```

**Comment by Luthaf** [comment]
Closed in favor of #211
